docs/install.md

```diff
@@ -406,6 +406,100 @@ Select the following tabs for the installation description of your target.
     <font size="+1">**Generate a Hyper-V Generation 2 machine**</font>  
     An option to get a Hyper-V Generation 2 machine is to generate your own Hyper-V image via a **Debian network installation** (booting the Hyper-V machine from a Debian `netinst.iso` installer like you would do it on a PC). Install a minimal Debian machine (i.e. no X11 desktops, etc.). Afterwards run the procedure described in section ["Make your own distribution"](../hardware/#make-your-own-distribution). Generation 2 machines support (and require) to boot in UEFI mode, support [Secure Boot](https://en.wikipedia.org/wiki/Unified_Extensible_Firmware_Interface#Secure_Boot), [TPM](https://en.wikipedia.org/wiki/Trusted_Platform_Module), use modern SCSI controllers and have higher hardware limits. For use as a home server, however, you will not need any of these functions.
 
+=== "Install in Parallels (Mac OS)"
```

Comment:
Not directly related to this PR, but what do you think about shortening the tab names, i.e. `Parallels (macOS)`, `VMware`, `VirtualBox`, skipping the `Install in/on`? Would enhance overview on small screens, also or especially once we add the scrollable tab style.

Reply:
Ok. Done.